### PR TITLE
JS exclusions for Mailerlite forms

### LINKED ISSUE #2065: JS exclusions for Mailerlite forms
When using the official [Mailerlite plugin](https://wordpress.org/plugins/official-mailerlite-sign-up-forms/) to add embedded forms, there are some exclusions we have to make for Deferred JS and Combine JS, so that the forms show up. 

**Combine JS**

Exclude Inline JS: 
`static.mailerlite.com`

Excluded JS file:
`app.mailerlite.com`

**Deferred JS**
Exclude: `app.mailerlite.com`

These exclusions have been tested and confirmed in the UI and with a helper on a customer's site: https://secure.helpscout.net/conversation/982085982/127187/
But couldn't get the FTP access there in order to pass to Level 2. 

I have a test set up on mega and going to ping @vmanthos to do the validation and pull request

Tickets: https://secure.helpscout.net/search/?query=tag%3Amailerlite%20sign%20up%20forms

### PR DESCRIPTION
**Pull request by vmanthos**
Fixes #2065.

### CODE REVIEW

**Comment by Tabrisrp** [comment]
@vmanthos 2 things remaining to do:
- Update the unit/integration tests for them to pass
- Fix the remaining conflict between your branch & the develop branch

File: inc/functions/options.php [hunk 1]
```diff
@@ -413,6 +413,7 @@ function get_rocket_exclude_defer_js() {
 		'google.com/recaptcha',
 		'widget.reviews.co.uk',
 		'lib/admin/assets/lib/webfont/webfont.min.js'
```
Comment (crystinutzaa) [inline inc/functions/options.php]: Forgot to add a comma in here:
`'lib/admin/assets/lib/webfont/webfont.min.js',`

Comment (vmanthos) [reply inc/functions/options.php]: Oops! 😅 

Done!

**Comment by Tabrisrp** [comment]
There is 3 other tests that need to be updated:
- one in `/tests/Unit/Functions/Options/TestExcludeDeferJS.php`
- two in `/tests/Integration/Functions/Options/TestExcludeDeferJS.php`

You can look at the Job on Travis to see why they're failing, it's expecting an array with specific values, and the values received are different

**Pull request merged**